code/datums/pod_style.dm

```diff
@@ -0,0 +1,140 @@
+
+/datum/pod_style
+	var/name = "supply pod"
+	var/ui_name = "Standard"
+	var/desc = "A Nanotrasen supply drop pod"
+	var/shape = POD_SHAPE_NORMAL
+	var/icon_state = "pod"
+	var/has_door = TRUE
+	var/decal_icon = "default"
+	var/glow_color = "yellow"
+	var/rubble_type = RUBBLE_NORMAL
+	var/id = "standard"
```

Comment:
This could use a class level doc saying what it's used for

And also all these comments could use some documentation as well